conda-store-server/conda_store_server/orm.py

```diff
@@ -173,6 +174,9 @@ class Build(Base):
     package_builds = relationship("CondaPackageBuild", secondary=build_conda_package)
 
     status = Column(Enum(schema.BuildStatus), default=schema.BuildStatus.QUEUED)
+    # Additional status info that will be provided to the user. DO NOT put
+    # sensitive data here
+    status_info = Column(UnicodeText, default=None)
```

Comment:
Any maximum length recommended?

Reply:
SQLAlchemy doesn't makes any promises about this type: https://docs.sqlalchemy.org/en/20/core/type_basics.html#sqlalchemy.types.UnicodeText. I think it depends on your database backend, so I'm not making any promises here. I also feel like it's not something worth pointing out next to every field. This is not the only place where we use UnicodeText, so it's not something special either (see `Environment.description`).